airbyte-webapp/src/views/Connection/ConnectionForm/formConfig.test.ts

```diff
@@ -40,7 +40,8 @@ describe("#useFrequencyDropdownData", () => {
     };
     const { result } = renderHook(() => useFrequencyDropdownData(additionalFrequency), { wrapper });
 
-    expect(result.current.length).toEqual(frequencyConfig.length + 1);
+    // +2 for cron and manual frequencies
+    expect(result.current.length).toEqual(frequencyConfig.length + 1 + 2);
```

Comment:
This is +3

Reply:
I just wrote it that way for better view but yeah can change to +3

Reply:
No the comment says +2 but the test is +3.

Reply:
I think the +1 is for a custom frequency, since this test is checking that a custom frequency (e.g. maybe created through the API) is shown in the dropdown if it doesn't match any of the default ones

Reply:
Yeah true